packages/analytics-v1.1/src/features/core/metrics/errorReporting/providers/Bugsnag.js

```diff
@@ -121,10 +121,11 @@ class BugsnagProvider {
    * If already loaded initialize the SDK.
    */
   init() {
-    // Return if RS Bugsnag instance is already initialized
+    // Return if RS Bugsnag instance is already initialized or should not init
     if (
       window.RudderStackGlobals &&
-      window.RudderStackGlobals[ERROR_REPORTING_SERVICE_GLOBAL_KEY_NAME]
+      window.RudderStackGlobals[ERROR_REPORTING_SERVICE_GLOBAL_KEY_NAME] &&
+      BUGSNAG_CDN_URL
```

Comment:
Did you mean this?
We should return if RS Bugsnag is already initialized or the CDN URL is invalid
```suggestion
      (window.RudderStackGlobals &&
      window.RudderStackGlobals[ERROR_REPORTING_SERVICE_GLOBAL_KEY_NAME]) || (
      !BUGSNAG_CDN_URL)
```

Reply:
probably, let me check this one more

Reply:
fixed